docs/guides/4-custom-rulesets.md

```diff
@@ -396,6 +396,48 @@ aliases:
 
 Rulesets can then reference aliases in the [given](#given) keyword, either in full: `"given": "#Paths"`, or use it as a prefix for further JSONPath syntax, like dot notation: `"given": "#ParameterObject.name"`.
 
+Bear in mind that an alias has to be explicitly defined in either at the top-level or inside an override.
+This is to avoid ambiguity.
+
+```yaml
+aliases:
+  Stoplight:
+    - "$..stoplight"
+overrides:
+  - files:
+      - "*.yaml"
+    rules:
+      value-matches-stoplight:
+        message: Value must contain Stoplight
+        given: "#Stoplight" # valid because declared at the root
+        severity: error
+        then:
+          field: description
+          function: pattern
+          functionOptions:
+            match: Stoplight
+  - files:
+      - "**/*.json"
+    aliases:
+      Value:
+        - "$..value"
```

Comment:
Might be worth adding a note re override behavior. E.g. - if an alias called `Stoplight` were defined here, would it take precedence over the `Stoplight` alias defined in the root?